homework-submission/myapp/src/stores/rashadStore.js

```diff
@@ -0,0 +1,38 @@
+import { observable, action, computed, configure } from "mobx";
+import Todos from "../components/todos.json";
+
+configure({ enforceActions: true });
+
+class RashadStore {
+  @observable todos = Todos;
+
+  @action
+  checker = item => {
+    item.done = !item.done;
+  };
+
+  @action
+  remover = item => {
+    const remove = this.todos.filter(elem => elem !== item);
+    this.todos = remove;
+  };
+
+  @action
+  submitHandler = e => {
+    e.preventDefault();
+    const newTodo = {
+      id: this.todos.length + 1,
```

Comment:
the Idea of `id` is to set a unique property for the object!
So the length in sommepoint won't be unique think about this:
>you have 3 items the length is 3 ids: 1, 2, 3
>you deleted a one in the middle so your ids now: 1, 3
>you added a new one your ids: 1, 3, 3

see? 🙃